compiler/noirc_evaluator/src/ssa/interpreter/value.rs

```diff
@@ -129,6 +129,13 @@ impl Value {
         Self::Numeric(NumericValue::from_constant(constant, typ))
     }
 
+    pub(crate) fn from_slice(slice: &[FieldElement], typ: NumericType) -> Self {
```

Comment:
Can you name this "array_from_slice" or similar to make it clear we're making an array and not a slice value?